crates/matrix-sdk-ui/src/sync_service/mod.rs

```diff
@@ -319,11 +337,10 @@ impl SyncService {
     /// This must be called when the app goes into the background. It's better
     /// to call this API when the application exits, although not strictly
     /// necessary.
+    #[instrument(skip_all)]
     pub async fn pause(&self) -> Result<(), Error> {
```

Comment:
Do we rename it `stop`?

Reply:
Oh that's been done in a subsequent commit :+1: